country-dashboard/src/index.tsx

```diff
@@ -3,13 +3,21 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+// routing
+const router=createBrowserRouter([{
+  path:"/",
+  element:<App/>
```

Comment:
I appreciate that you set up routing even though it was not required, +1 bonus point